src/components/FacebookFeed.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+export class FacebookFeed extends React.Component<any, any> {
```

Comment:
Declaring an interface for props and state helps insure type consistency if used outside the class (tests, other components). I like to declare the props and state in the same file because they are strongly related to the component, but this is just preference!

```suggestion
export interface FacebookFeedState {
    width: number
    height: number
}

export class FacebookFeed extends React.Component<any, FacebookFeedState> {

```

Reply:
No problem with that!